index.html

```diff
@@ -204,55 +204,1951 @@ <h3>
+              6.2.2.6. An item with the <dfn>Input tag</dfn>, <dfn>Output
+              tag</dfn>, or <dfn>Feature tag</dfn> defines a field within an
+              input, output, or feature report. The [=item data field=] of a
+              [=Main item=] with an [=Input tag=], [=Output tag=], or [=Feature
+              tag=] is a bit field containing several boolean parameters
+              describing the data field. An item with the <dfn>Collection
+              tag</dfn> defines a new collection nested within the current
+              collection, or a new [=top-level collection=] if there is no
+              current collection. The [=item data field=] of a [=Main item=]
+              with a [=Collection tag=] contains the <dfn>collection
+              type</dfn>. An item with the <dfn>End Collection tag</dfn> closes
+              the current collection.
+            </p>
+          </div>
+          <ol>
+            <li>If |item| is a [=Main item=]:
+              <ol>
+                <li>If |item| has the [=Input tag=], [=Output tag=], or
+                [=Feature tag=]:
+                  <ol>
+                    <li>Let |globalState| be the last item of |globalStack|.
+                    </li>
+                    <li>Let |reportId:octet| be |globalState|[`"reportId"`].
+                    </li>
+                    <li>Let |reportItem:HIDReportItem| be the result of
+                    [=creating a HID report item=] with |item|, |localState|,
+                    |globalState|, |usages|, and |strings|.
+                    </li>
+                    <li>For each |collection:HIDCollectionInfo| of |ancestors|:
+                      <ol>
+                        <li>Let |reports:sequence&lt;HIDCollectionInfo&gt;|
+                        be:<br>
+                          |collection|.{{HIDCollectionInfo/inputReports}} if
+                          |item| has the [=Input tag=],<br>
+                          |collection|.{{HIDCollectionInfo/outputReports}} if
+                          |item| has the [=Output tag=], or<br>
+                          |collection|.{{HIDCollectionInfo/featureReports}} if
+                          |item| has the [=Feature tag=].
+                        </li>
+                        <li>
+                          <p>
+                            If |reports| contains an item with its
+                            {{HIDReportInfo/reportId}} attribute equal to
+                            |reportId|, then let |report| be that item.
+                          </p>
+                          <p>
+                            Otherwise:
+                          </p>
+                          <ol>
+                            <li>Let |report:HIDReportInfo| be a
+                            {{HIDReportInfo}}.
+                            </li>
+                            <li>Set |report|.{{HIDReportInfo/reportId}} to
```

Comment:
Because _report_ is a dictionary and not an interface the correct syntax for referring to its members is the same as when manipulating _initialGlobalState_ above.
```suggestion
                            <li>Set |report|["{{HIDReportInfo/reportId}}"] to
```

Reply:
Done